db/agents.json

```diff
@@ -588,13 +600,14 @@
       ]
     },
     {
-      "regex": "^Kids( |%20)Listen.+CFNetwork",
+      "regex": "^Kids Listen.+CFNetwork",
       "name": "Kids Listen",
       "type": "Mobile App",
       "os": "iOS",
       "examples": [
         "Kids%20Listen/55 CFNetwork/758.5.3 Darwin/15.6.0",
-        "Kids%20Listen/55 CFNetwork/902.2 Darwin/17.7.0"
+        "Kids%20Listen/55 CFNetwork/902.2 Darwin/17.7.0",
+        "Kids Listen/55 CFNetwork/902.2 Darwin/17.7.0"
```

Comment:
Trying to cover both cases now - fully decoded and this kind where the client has (maybe accidentally) url-encoded their app name but not the rest of the string.